CreatorNodeDocumentation/src/concepts/GeneralConcepts/ephemeralState.md

```diff
@@ -1,5 +1,5 @@
 # Ephemeral State
 
-Graphs have a boolean value representing their ephemeral state, and this state is exposed to graph logic through the **ephemeral** node.
+Graphs have a boolean value representing their ephemeral state, and this state is exposed to graph logic through the [**Ephemeral**](/nodes/Ephemeral/documentation.md). A graph is in an ephemeral state when parameters are actively being configured/adjusted. For example, when one is dragging a number slider in the Parameter panel.

```

Comment:
a graph *can be* in a ephemeral state...

it's driven programmatically via the SDK (including the CPV), so applications using the Materia service can put Materia in an ephemeral state ay time,